cybox/utils/__init__.py

```diff
@@ -73,6 +74,17 @@ def _import_submodules(pkg):
         importlib.import_module(mod_name)
 
 
+def is_sequence(item):
```

Comment:
I changed all of python-stix to use the `is_sequence` method because I didn't really like the restrictive nature of requiring a `list` for input. Users may want to pass in any (non-`Mapping`) iterable object such as `set`, `tuple`, or a generator object to extract values from. I guess I didn't see any reason to raise errors when instances of those objects are passed in.